openedx/core/djangoapps/content_libraries/rest_api/blocks.py

```diff
@@ -1,35 +1,34 @@
 """
 Content Library REST APIs related to XBlocks/Components and their static assets
 """
+import edx_api_doc_tools as apidocs
 from django.core.exceptions import ObjectDoesNotExist
 from django.db.transaction import non_atomic_requests
 from django.http import Http404, HttpResponse, StreamingHttpResponse
 from django.urls import reverse
 from django.utils.decorators import method_decorator
 from drf_yasg.utils import swagger_auto_schema
+from opaque_keys.edx.locator import LibraryLocatorV2, LibraryUsageLocatorV2
+from openedx_learning.api import authoring as authoring_api
 from rest_framework import status
 from rest_framework.exceptions import NotFound, ValidationError
 from rest_framework.generics import GenericAPIView
 from rest_framework.parsers import MultiPartParser
 from rest_framework.response import Response
 from rest_framework.views import APIView
 
-import edx_api_doc_tools as apidocs
-from opaque_keys.edx.locator import LibraryLocatorV2, LibraryUsageLocatorV2
-from openedx_learning.api import authoring as authoring_api
-
 from openedx.core.djangoapps.content_libraries import api, permissions
 from openedx.core.djangoapps.content_libraries.rest_api.serializers import (
-    ContentLibraryComponentCollectionsUpdateSerializer,
+    ContentLibraryItemCollectionsUpdateSerializer,
     LibraryXBlockCreationSerializer,
     LibraryXBlockMetadataSerializer,
     LibraryXBlockOlxSerializer,
     LibraryXBlockStaticFileSerializer,
-    LibraryXBlockStaticFilesSerializer,
+    LibraryXBlockStaticFilesSerializer
```

Comment:
nit: should keep trailing `,`
```suggestion
    LibraryXBlockStaticFilesSerializer,
```